app/views/content/terms_candidate.md

```diff
@@ -43,3 +56,21 @@ All candidates must have an enhanced DBS check. This will show up any spent or u
 Not all convictions will stop you from teaching. [Talk to your provider about their policy on criminal convictions](https://www.gov.uk/exoffenders-and-employment).
 
 Your name will also be checked against a DBS list of people who have been barred from working with children.
+
+## Questions or support
+
+If you need our help, email <becomingateacher@digital.education.gov.uk>.
+
+We respond within 5 working days, or 1 working day for more urgent queries.
+
+If you’re applying from overseas, you can find out more about becoming a teacher in England in our guidance for overseas graduates.
+
+## Changing our service
+
+Our service is always evolving, and we will make changes from time to time. We won’t tell you about the changes beforehand. We will let you know if the website will be unavailable while we’re updating it.
```

Comment:
We should avoid negative contractions for consistency.

Change 'won't' to 'will not'.